proposed/phpdoc-tags.md

```diff
@@ -368,7 +368,7 @@ Mark a function that has been generated using a PHP script.
 /**
  * @generated function generated using bin/script.php, please DO NOT EDIT!
  * 
- * @params array $params parameters for the index API endpoint
+ * @param array $params parameters for the index API endpoint
```

Comment:
Good catch. This is a typo and should be fixed separately.

Reply:
> 接得好。這是一個拼字錯誤，應單獨修復。